### PR TITLE
Hamburger menu

### PR DESCRIPTION
**Pull request by arun-kushwaha04**
## Fixes Issue
Added feature requested in issue #227 

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots
image
image

### CODE REVIEW

File: src/lib/components/header.svelte [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
   import ProfilePicture from './profile-picture.svelte';
   import { page } from '$app/stores';
+  import { onMount } from 'svelte';
   export let username: string;
 
   const navItems = [{ name: 'Docs', path: '/docs' }];
+  let closeHamburgerMenu: () => void;
+
+  onMount(() => {
+    const burger = document.querySelector('.navbar-burger');
```
Comment (Cahllagerfeld) [inline src/lib/components/header.svelte:11]: I think you can use `bind:this` to bind an HTMLElement. You don't need a querySelector here 🤔

**Review by Cahllagerfeld — COMMENTED** [review]
One small comment. On a sidenote the tests in the pipeline are failing, could you please fix them as well 🙏

File: src/lib/components/header.svelte (hunk 1 above)
Comment (arun-kushwaha04) [reply src/lib/components/header.svelte]: Sure, did the changes.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -69,9 +69,9 @@ git merge upstream/main
```
Comment (Cahllagerfeld) [inline CONTRIBUTING.md:69]: Could you please revert the changes in that file 🙏

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]
Light-Mode: 
grafik
could you please adjust the icon for light-mode. Currently the SVG isn't visible in lightmode

grafik
Although light-mode is selected, the panel that opens is in darkmode and not all menu items are readable

**Comment by arun-kushwaha04** [comment]
image
@Cahllagerfeld fixed the hamburger in light mode

**Review by Cahllagerfeld — COMMENTED** [review]
Nice work! I think the last step missing is reverting the changes from `Contributing.md`. Then we are good to go 🙏

File: CONTRIBUTING.md (hunk 2 above)
Comment (arun-kushwaha04) [reply CONTRIBUTING.md]: @Cahllagerfeld , reverted `CONTRIBUTING.md` to `29f3726f70a3815687d5d4b84d6880945aef34be`

**Review by Cahllagerfeld — APPROVED** [review]
NIce work!

**Comment by arun-kushwaha04** [comment]
> NIce work!
> 
> 2 tests are still running red though, do you know how to fix them?

@Cahllagerfeld , found the issue. I added new test for mobile devices as the old test code was unable to execute the test.
